### PR TITLE
issue-1-create-an-entry

### PR DESCRIPTION
**Pull request by gino-hong**
Large desktop-sized screen
Screenshot 2022-03-29 190821
Small desktop-sized screen
Screenshot 2022-03-29 191017
Mobile screen
Screenshot 2022-03-29 191052

### CODE REVIEW

**Comment by gino-hong** [comment]
issue 1 demo

File: js/data.js [hunk 1]
```diff
@@ -6,3 +6,29 @@ var data = {
...
+  var object = {};
+  object.title = $form.elements.title.value;
+  object.photoUrl = $form.elements.photoUrl.value;
+  object.notes = $form.elements.notes.value;
+  object.entryId = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.push(object);
+  $form.reset();
+  $image.src = 'images/placeholder-image-square.jpg';
+});
```
Comment (BrettAlbright) [inline js/data.js]: The form functionality should be in `main.js` the only functionality you want in `data.js` is the local storage functionality and main data object.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Looks great! Just a little code to move

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,32 @@
...
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var object = {};
+  object.title = $form.elements.title.value;
+  object.photoUrl = $form.elements.photoUrl.value;
+  object.notes = $form.elements.notes.value;
+  object.entryId = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.push(object);
```
Comment (BrettAlbright) [inline js/main.js]: you want to use `unShift()` here because when you render the entries in the next feature, you want to put the newest ones at the top.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
tiny update!

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**